orderapi/src/main/java/fi/hel/verkkokauppa/order/service/accounting/AccountingSlipService.java

```diff
@@ -68,7 +68,7 @@ public class AccountingSlipService {
     @Autowired
     private AccountingSearchService accountingSearchService;
 
-    public List<AccountingSlipDto> createAccountingData() {
+    public List<AccountingSlipDto> createAccountingData() throws Exception {
```

Comment:
where is this exception handled?

Reply:
It is handled in AccountingController.createAccountingData()

Reply:
log.error("creating accounting data failed", e);
            throw new CommonApiException(
                    HttpStatus.INTERNAL_SERVER_ERROR,
                    new Error("failed-to-create-accounting-data", "failed to create accounting data")
            );